Segmentation fault in constructObject function

**Description**
When installing istio, the install fails with the following error
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x16d84be]

goroutine 14 [running]:
github.com/layer5io/meshkit/utils/kubernetes.constructObject(0x22c0620, 0xc0003c51e0, 0xc000aba2c0, 0x17, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/utkarsh/go/pkg/mod/github.com/layer5io/meshkit@v0.1.31/utils/kubernetes/apply-manifest.go:107 +0xbe
github.com/layer5io/meshkit/utils/kubernetes.(*Client).ApplyManifest(0xc0005ebc00, 0xc0004a63c0, 0xef, 0xf0, 0x0, 0x0, 0x7f7f6c180001, 0xc0008878b0, 0x0)
        /home/utkarsh/go/pkg/mod/github.com/layer5io/meshkit@v0.1.31/utils/kubernetes/apply-manifest.go:48 +0x1d1
github.com/layer5io/meshery-istio/istio.(*Istio).applyManifest(0xc0000f4000, 0xc0004a63c0, 0xef, 0xf0, 0x0, 0x0, 0x0, 0x0, 0x4161fd)
        /home/utkarsh/dev/layer5/meshery-istio/istio/install.go:98 +0x136
github.com/layer5io/meshery-istio/istio.(*Istio).installIstio(0xc0000f4000, 0xc000a0b500, 0xc0000cb618, 0x5, 0xc0000cb000, 0x7, 0xc0007fc284, 0x7f7f96a5e7d0, 0xc00092d8c0, 0xc000073740)
        /home/utkarsh/dev/layer5/meshery-istio/istio/install.go:52 +0x534
github.com/layer5io/meshery-istio/istio.(*Istio).ApplyOperation.func1(0xc0005bc0a0, 0xc0000caff8, 0x5, 0xc0000cb000, 0x7, 0xc0000cb010, 0xb, 0x0, 0x0, 0x0, ...)
        /home/utkarsh/dev/layer5/meshery-istio/istio/istio.go:50 +0xd1
created by github.com/layer5io/meshery-istio/istio.(*Istio).ApplyOperation
        /home/utkarsh/dev/layer5/meshery-istio/istio/istio.go:48 +0x277
exit status 2
```

**Expected Behavior**
Installation should succeed without any issues.

**Environment:**
- Meshkit - v0.1.31

---
**To Reproduce**
Steps to reproduce the behavior:
1. Get the latest version of istio adapter
2. Install istio from the meshery ui

--- a/utils/kubernetes/apply-manifest.go
+++ b/utils/kubernetes/apply-manifest.go
@@ -40,8 +40,11 @@ func (client *Client) ApplyManifest(contents []byte, recvOptions ApplyOptions) e
 		options := recvOptions
 
 		// get the runtime.Object
-		object, _, err := client.GetObjectFromManifest(manifest)
+		object, obj, err := client.GetObjectFromManifest(manifest)
 		if err != nil {
+			if len(obj.GetObjectKind().GroupVersionKind().Kind) < 1 {
+				continue
+			}
 			return ErrApplyManifest(err)
 		}
 
@@ -89,8 +92,8 @@ func (client *Client) GetObjectFromManifest(manifest string) (runtime.Object, *u
 	obj := &unstructured.Unstructured{}
 	dec := yaml.NewDecodingSerializer(unstructured.UnstructuredJSONScheme)
 	object, _, err := dec.Decode([]byte(manifest), nil, obj)
-	if err != nil && len(obj.GetObjectKind().GroupVersionKind().Kind) > 1 {
-		return nil, nil, ErrApplyManifest(err)
+	if err != nil {
+		return nil, obj, ErrApplyManifest(err)
 	}
 	return object, obj, nil
 }

